backend/app/treasury/tests/test_make_transfers_for_treasury.py

```diff
@@ -5,13 +5,14 @@
 from unittest.mock import Mock
 
 import pytest
+import pytest_asyncio
 from pytz import UTC
 
 from .. import actions
 from ..models import ERC20, Transfer, Treasury
 
 
-@pytest.fixture
+@pytest_asyncio.fixture
```

Comment:
:+1:

Reply:
> Next challenges I see:
> 
> * `treasury` should not export its own adapters. One approach could be that Celery tasks are defined in `treasury/tasks.py` and somehow registered inside the global `celery_app`.
> * as "adapters" are getting more complex, we might need to look into splitting down (probably adapters vs ports) more intelligently so as to play better with our caching logic.

Interesting, I'll definitely like to continue with this conversation.